src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+const listItems = [...document.querySelectorAll('li')];
+const employeesList = document.querySelector('ul');
+
+function convertToNumber(string) {
+  const salaryNumber = string.replace(/\$/g, '').replace(/,/g, '');
+
+  return +salaryNumber;
```

Comment:
The convertToNumber function converts the string to a number using the unary plus operator. This can cause problems if the string cannot be converted to a number. It's better to use the Number constructor which returns NaN if the conversion fails.